__tests__/get.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app.js");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection.js");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe.only("bad paths", () => {
+  test("GET 404 return 404 on invalid url", () => {
+    return request(app)
+      .get("/api/wrongURL")
+      .expect(404)
+      .then((result) => {
+        expect(result.body.message).toBe("invalid url");
+      });
+  });
+});
+
+describe("GET", () => {
+  describe("GET api/categories", () => {
+    test("return the categories ", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((result) => {
+          expect(result.body.categories).toBeInstanceOf(Array);
+          result.body.categories.forEach((category) => {
```

Comment:
Good use of a .forEach here, but you'll need to check that the array isn't empty first. Otherwise, the .forEach will be skipped and you'll get an automatic test passing.